### PR TITLE
CI: Adding Starlight to Docker

### PR DESCRIPTION
**Pull request by chexware**
Added Dockerfile configuration for Starlight

### CODE REVIEW

**Comment by girazoki** [comment]
Update the description of the PR please

File: docker/starlight.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	rm -rf /usr/sbin
+
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+USER tanssi-relay
+
+COPY --chown=tanssi-relay build/tanssi-relay* /tanssi-relay
+RUN chmod uog+x /tanssi-relay/tanssi-relay*
+
+# 30333 for parachain p2p
```
Comment (tmpolaczyk) [inline docker/starlight.Dockerfile:27]: This port is not needed, also these are the defaults of `tanssi-node`, they may be different in `tanssi-relay`

Comment (girazoki) [reply docker/starlight.Dockerfile:27]: I think by default we need to expose: 30333 9933 9944 9615. Based of https://github.com/paritytech/polkadot-sdk/blob/master/substrate/scripts/ci/docker/substrate.Dockerfile

Comment (girazoki) [reply docker/starlight.Dockerfile:27]: AND https://github.com/paritytech/polkadot-sdk/blob/master/docker/dockerfiles/polkadot/polkadot_builder.Dockerfile

Comment (chexware) [reply docker/starlight.Dockerfile]: Updated in https://github.com/moondance-labs/tanssi/pull/698/commits/648e4b9cad9922a8972c3cc81740afae6764b732

**Review by tmpolaczyk — APPROVED** [review]

**Review by girazoki — APPROVED** [review]

**Pull request merged**